lib/quark/compose.ex

```diff
@@ -73,7 +73,7 @@ defmodule Quark.Compose do
   def g <|> f, do: compose(g, f)
 
   @doc ~S"""
-  Function composition, from the back of the lift to the front
+  Function composition, from the back of the list to the front
```

Comment:
'lift' does not really seem proper in this context to me either, I really think it was a mis-copy/paste?  :-/